app/src/main/java/ch/epfl/sweng/swenggolf/offer/create/CreateListeners.java

```diff
@@ -103,7 +110,7 @@ public void onComplete(DbError error) {
      *
      * @return a listener
      */
-    public ValueListener<Offer.Builder> restoreOfferListener() {
+     ValueListener<Offer.Builder> restoreOfferListener() {
```

Comment:
Why did you decide to remove the public?

Reply:
CreateListener methods are only used in this package, so I think it is better if we can only access it here.